GitVersion.yml

```diff
@@ -3,5 +3,7 @@ mode: ContinuousDeployment
 continuous-delivery-fallback-tag: preview
 
 branches:
+  master:
+    regex: ^master$|^main$
```

Comment:
needed to amend the default configuration of GitVersion so to support `main` as new master branch.